jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/registry/installations/InstallationRegistrationEndpoint.java

```diff
@@ -73,7 +78,7 @@ public Response crossOriginForInstallations(@Context HttpHeaders headers) {
      *
      * <pre>
      * curl -u "variantID:secret"
-     *   -v -H "Accept: application/json" -H "Content-type: application/json"
+     *   -v -H "Accept: application/json" -H "Content-type: application/json" -H "aerogear-push-id: someid"
```

Comment:
Will be added in our Registration doc https://aerogear.org/docs/unifiedpush/client-registration/ , seems that the good place ?